.github/workflows/release.yml

```
@@ -0,0 +1,24 @@
+name: Release the gem
+
+on:
+  create:
+    ref_type: tag
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'voxpupuli/beaker'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - name: Build gem
+        uses: scarhand/actions-ruby@master
+        with:
+          args: build *.gemspec
+      - name: Publish gem
+        uses: scarhand/actions-ruby@master
+        env:
+          RUBYGEMS_AUTH_TOKEN: ${{ secrets.RUBYGEMS_AUTH_TOKEN }}
+        with:
+          args: push *.gem
```

Given how minimal this container image is, does it make sense to just use the raw ruby container and execute the shell commands manually? The reason I'm asking is mostly security: this container could be modified to leak the auth token and it would be hard to detect this.

Untested, but perhaps the following gives more flexibility?

```suggestion
      - name: Install Ruby 2.6
        uses: ruby/setup-ruby@v1
        with:
          ruby-version: '2.6'
      - name: Build gem
        shell: gem build *.gemspec
      - name: Setup credentials
        shell: |
          mkdir -p ~/.gem
          echo -e "---\n:rubygems_api_key: ${{ secrets.RUBYGEMS_AUTH_TOKEN }}" > ~/.gem/credentials
          chmod 0600 ~/.gem/credentials
      - name: Publish gem
        shell: gem push *.gem
```